cockroachdb/templates/poddisruptionbudget.yaml

```
@@ -1,5 +1,5 @@
 kind: PodDisruptionBudget
-{{- if and (.Capabilities.APIVersions.Has "policy/v1") (semverCompare ">= 1.21-0" .Capabilities.KubeVersion.Version) }}
+{{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion }}
```

I believe this condition is failing because of the extra space in ">= 1.21-0". Once we remove the extra space the existing condition should pass with ">=1.21-0". Also, instead of `anding` these two conditions, we should use `or` operator to give a little extra cushion. So final condition should be:
```suggestion
{{- if or (.Capabilities.APIVersions.Has "policy/v1") (semverCompare ">=1.21-0" .Capabilities.KubeVersion.Version) }}
```

Can you please try with this condition and let me know if it passes on AKS.

---

Yes, works that way.